app/routes/v2/clusters.js

```diff
@@ -183,6 +183,8 @@ const syncClusterResources = async(req, res)=>{
   res.status(200).send('Thanks');
 };
 
+
+// todo: deprecate this after forcing updater to use the graphql/v3 endpoint
```

Comment:
The overall intent is not clear.  The implementation implication is to replace `POST /v2/:cluster_id/resources` with `PUT /v3/:cluster_id/resources`.  However:
- `/v2` APIs use org-key authentication while GQL and `/v3` APIs use user bearer token IIRC.
- There isn't a plan to deprecate `/v2` APIs and update all `watch-keeper` instances to use the `/v3` replacement and different auth.

Wouldn't it be better to simply update the existing `/v2` cluster resources implementation to call GQL instead?